test/async-hooks/test-improper-order.js

```diff
@@ -51,8 +51,12 @@ if (process.argv[2] === 'child') {
   child.stderr.on('data', (d) => { errData = Buffer.concat([ errData, d ]); });
   child.stdout.on('data', (d) => { outData = Buffer.concat([ outData, d ]); });
 
-  child.on('close', common.mustCall((code) => {
-    assert.strictEqual(code, 1);
+  child.on('close', common.mustCall((code, signal) => {
+     if (signal) {
```

Comment:
```suggestion
    if (signal) {
```
This should fix the formatting